lib/graphics.js

```diff
@@ -375,6 +378,35 @@ graphics.prototype.drawControls = function() {
 
 }
 
+graphics.prototype.getImagesForPincode = function(pincode) {
+	var self = this;
+	var b = "1 2 3 4 6 7 8 9 11 12 13 14".split(/ /);
+
+	if (self.pincodebuffer[0] === undefined) {
+		var img;
```

Comment:
This should move up to the declarations at 382-383.  Otherwise it may not be declared when lines 396 and 403 execute given its inside the if.

Reply:
Noted, it's been moved